CHANGELOG.md

```diff
@@ -18,6 +18,14 @@
 
 ---
 
+## v5.31.2
+
+**Public Changes:**
+
+- ✨ [RUM-7371] move React integration out of beta ([#3160](https://github.com/DataDog/browser-sdk/pull/3160)) [RUM] [RUM-REACT] [RUM-SLIM]
```

Comment:
**💬 suggestion:** ‏because this is a new public feature, the version should be `v5.32.0`